tests/evaluators/test_content_exclusion.py

```diff
@@ -0,0 +1,78 @@
+from agentsim import Simulator
+from agentsim.utils import llm_eval_call, check_guardrail
+from agentsim.evaluation import evaluator
+
+# YAML config for the Content Exclusion evaluator
+config = """
+---
+evaluators:
+    content_exclusion:
+        model_settings:
+            model: openai/gpt-4o
+            api_key: OPENAI_API_KEY
+            system_prompt: >
+                [Instruction]
+                Please act as an impartial judge and evaluate the quality of the output to the input displayed below. Your evaluation should be based on the mentioned criteria. Begin your evaluation by providing a short explanation on how the output performs relative to the input. Be as objective as possible. After providing your explanation, you must rate the response on a scale of 1 to 5 by strictly following this JSON format: '{"rating": rating}, for example: '{"rating": 2}.
+                [Criteria]
+
+                Check if the response correctly omits the specified elements or topics.
+
+                [The Start of Input]
+                {{messages}}
+                [The End of Input]
+                [Evaluation With Rating in JSON format]
+            json_mode: on
+        target: '[4,5]' # target score range
+        in_range: numeric_range
```

Comment:
add a templated param called {{topics_to_omit}} in the input section. Modify the prompt to check if the messages omit the topics_to_omit. Describe the scoring criteria in more detail. Qualitatively describe the low and high scores.